.github/workflows/pr-handling.yaml

```diff
@@ -0,0 +1,38 @@
+name: Assign PR to author and reviewers
+
+# This workflow runs on the pull_request_target event:
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
+#
+# So the secrets are accessible even if the PR was opened from an external
+# repository. This is done because the GitHub API key needs to be accessed
+# to modify the PRs.
+#
+# NOTE: Only API calls should be made in the actions defined here. The
+#       committed code should _NOT_ be touched in any case.
+
+on:
+  pull_request_target:
+    types: [ opened, reopened ]
+
+jobs:
+  assign-pr:
+    name: Assign PR to author
+    runs-on: ubuntu-latest
+    steps:
+      - uses: toshimaru/auto-author-assign@v1.6.0
```

Comment:
kindly keep in mind that v1.6.1 is out - however from the Changelog, no security fixes are evident. Please consider keeping this up-2-date with the releases.

Reply:
Thank you for pointing this out. The action is updated.